pages/blog/swr-v2.en-US.mdx

```diff
@@ -0,0 +1,278 @@
+---
+image:
+description: 'We are excited to introduce SWR v2.0, including features to improve the use experience of your application.'
+date:
+---
+
+import Callout from 'nextra-theme-docs/callout'
+import Bleed from 'nextra-theme-docs/bleed'
+
+import Authors, { Author } from 'components/authors'
+import Video from 'components/video'
+
+# Announcing SWR 2.0
+
+<Authors date="">
+</Authors>
+
+We are excited to introduce SWR v2.0, including features to improve the use experience of your application.
+
+## What’s New
+
+### Better React 18 Support
+
+SWR now uses `useSyncExternalStore` and `startTransition` internally to improve React 18 support, which ensures stronger UI consistency in concurrent rendering with React 18. The change is an internal implementation detail, so developers using SWR do not have to care about the change.
+
+### isLoading State
+
+`isLoading` is a new return value that indicates if there's an ongoing request and no "loaded data". Previously, you have to check `data` and `error` are `undefined`, but now you can use the `isLoading` value.
```

Comment:
Since previously users can also use `isValidating`, we might want to clarify what's the difference.

Reply:
@huozhi Thank you! I'll create a new section for the difference between `isLoading` and `isValidating` on the Understanding SWR page and put the link in the blog post.